sbhs/templates/webcam/show_video.html

```diff
@@ -27,11 +27,11 @@ <h3>SBHS live feed: Machine ID {{mid}}</h3>
 			</div>
 		</div>
 	</div>
-	<script type="text/javascript">
+	<script>
 		{% if request.user.profile.is_moderator %}
 			setInterval(function(){
 				var refresh = new Image();
-				refresh.src = "{% static  image_link %}";
+				refresh.src = "{% static image_link %}";
```

Comment:
A suggestion, the function defined here can be defined outside if tag as the same operation is done in else!
So you can simply use the function defined outside if, wherever needed!